cdk/lib/core/core-service-web-acl.ts

```diff
@@ -0,0 +1,45 @@
+import { CfnWebACL, CfnWebACLAssociation } from 'aws-cdk-lib/aws-wafv2';
+import { Construct } from 'constructs';
+
+export class CoreServiceWebACL extends Construct {
+  readonly webACL: CfnWebACL;
+
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    this.webACL = new CfnWebACL(this, 'WebACL', {
+      defaultAction: { allow: {} },
+      scope: 'REGIONAL',
+      visibilityConfig: {
+        cloudWatchMetricsEnabled: false,
+        metricName: 'IoTAppCoreServiceWebACL',
+        sampledRequestsEnabled: true,
+      },
+      rules: [
+        {
+          action: { block: {} },
+          name: 'IoTAppCoreServiceRateLimit',
+          priority: 0,
+          statement: {
+            rateBasedStatement: {
+              aggregateKeyType: 'IP',
+              limit: 3000, // 10 TPS over 5 minutes
```

Comment:
How does someone determine what the limit should be?

Reply:
> Looks good. Is there an easy way we can test this to make sure it works? Some script that hits the dashboard endpoint at over 10 TPS?

> How does someone determine what the limit should be?

We will be performing load testing as part of our operational readiness review
The artifact can help us and customers to determine the limit 👍